test-infra/ansible/ci-runner/Dockerfile

```
@@ -0,0 +1,26 @@
+    curl jq git build-essential \
+    libssl-dev libffi-dev \
+    ca-certificates gnupg lsb-release \
+    libdigest-sha-perl \
+    && rm -rf /var/lib/apt/lists/*
+RUN mkdir -p /etc/apt/keyrings && \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg && \
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    apt-get update && apt-get install -y docker-ce-cli
+WORKDIR /actions-runner
+ARG GH_TOKEN
+RUN LATEST_RELEASE=$(curl -s -H "Authorization: Bearer $GH_TOKEN" https://api.github.com/repos/actions/runner/releases/latest | jq -r .tag_name) && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)